### PR TITLE
domain tests

### PR DESCRIPTION
**Pull request by AlanE23**

### CODE REVIEW

**Comment by aoswalt** [comment]
:boom:

**Comment by Seeker1911** [comment]
change to domain

**Comment by josephkane** [comment]
🐗

File: test_domain.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import unittest
-from behavior import *
+from domain import *
 from tokenizer import *
 
 class TestBehavior(unittest.TestCase):
```
Comment (josephkane) [inline test_domain.py]: change this class please

**Comment by Seeker1911** [comment]
###Markdown
#markdown

**Comment by Seeker1911** [comment]
you missed one.

**Comment by Seeker1911** [comment]
👸

**Comment by josephkane** [comment]
🍏

**Pull request merged**